src/vsock/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "vhost-user-vsock"
+version = "0.1.0"
+authors = ["Harshavardhan Unnibhavi <harshanavkis@gmail.com>"]
+description = "A virtio-vsock device using the vhost-user protocol."
+repository = "https://github.com/rust-vmm/vhost-device"
+readme = "README.md"
+keywords = ["vhost", "vsock"]
+license = "Apache-2.0 OR BSD-3-Clause"
```

Comment:
Maybe add, edition = "2018", as well here ? I think that comes from the default values for this file. Just like the i2c part ?